### PR TITLE
DOC: add basic amplitude analysis tutorial

### PR DESCRIPTION
**Pull request by shenvitor**
- Closes https://github.com/ComPWA/RUB-EP1-AG/issues/93

> [!TIP]
> A temporary preview of this tutorial (**[TR-033](https://compwa.github.io/report/033)**) be found here [here](https://compwa--217.org.readthedocs.build/report/033.html).

> As a follow-up to https://github.com/ComPWA/RUB-EP1-AG/issues/91, it would be nice to formulate an amplitude model for this `Three-Particle-3.dat` data sample and try to fit it to exactly extract the parameters that were used to generate it. The goal of the exercise is to do a full amplitude analysis (with a simple model) purely with [NumPy ](https://numpy.org) and [SciPy](https://scipy.org), [`matplotlib`](https://matplotlib.org/), (~~optionally with the help of [`pylorentz`](https://pylorentz.rtfd.io)~~), [`vector`](https://vector.readthedocs.io/en/latest/index.html), [`phasespace`](https://github.com/zfit/phasespace), and [`iminuit`](https://iminuit.rtfd.io).

### CODE REVIEW

**Comment by shenvitor** [comment]
model_petapi

**Comment by shenvitor** [comment]
Something wrong when doing github action workflow at the moment.
see [here](https://github.com/ComPWA/compwa.github.io/actions/runs/8270205354/job/22627218012?pr=217)

seems like a problem related to Sphinx Extension Error in configuration of environment variables of colours (?)

> 
> Extension error (sphinx_codeautolink.extension):
> Handler <bound method SphinxCodeAutoLink.create_references of <sphinx_codeautolink.extension.SphinxCodeAutoLink object at 0x707c6999eb60>> for event 'env-updated' threw an exception (exception: invalid literal for int() with base 10: 'yes')
> docnb: exit 2 (134.03 seconds) /home/runner/work/compwa.github.io/compwa.github.io> sphinx-build --color --keep-going -TW -b html docs/ docs/_build/html pid=3114
>   docnb: FAIL code 2 (135.22=setup[1.19]+cmd[134.03] seconds)
>   evaluation failed :( (135.98 seconds)
> Error: Process completed with exit code 2.

**Comment by shenvitor** [comment]
Should I reopen this pull request?
Because the final words of the PWA101 documentation  is not yet finished 🤔

**Comment by shenvitor** [comment]
> > Should I reopen this pull request?
> 
> Oops sorry 😅 Hadn't realized I typed a ['fix' keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)...

Seems like the problem is still here after the fix attempt..
Probably need to check which cell later...

**Comment by shenvitor** [comment]
For the record, until this version: I have addressed most of @Smaldanerep1's comments.
Upcoming Next: @redeboer's turn 👀

**Comment by shenvitor** [comment]
A few broken links were reported in link check:
https://github.com/ComPWA/compwa.github.io/actions/runs/9501304513/job/26186513533?pr=217#step:5:447
https://github.com/ComPWA/compwa.github.io/actions/runs/9501304513/job/26186513533?pr=217#step:5:451
https://github.com/ComPWA/compwa.github.io/actions/runs/9501304513/job/26186513533?pr=217#step:5:453
https://github.com/ComPWA/compwa.github.io/actions/runs/9501304513/job/26186513533?pr=217#step:5:737

**Comment by shenvitor** [comment]
Wow 😱
according to: https://readthedocs.org/projects/compwa/builds/24685133/

Build #24685133
Pull Request #[217](https://github.com/ComPWA/compwa.github.io/pull/217) ([cbcd0d497ee7e99627154658c7dbedd1628f2e77](https://github.com/ComPWA/compwa.github.io/pull/217/commits/cbcd0d497ee7e99627154658c7dbedd1628f2e77))
Build failed
Error
The build was terminated due to time out. Read more about [time and memory limits in our documentation](https://docs.readthedocs.io/en/stable/builds.html#build-resources).

Maybe I should email them to apply: increase build limits on a per-project basis.
:)
Screenshot 2024-06-13 at 17 21 58

**Comment by redeboer** [comment]
No need, we just need to keep CI time small with smaller notebooks

**Review by redeboer — CHANGES_REQUESTED** [review]
@shenvitor, I'm quite happy with the notebook, looks nice!
I've written more comments at https://compwa--217.org.readthedocs.build/report/999.html. Didn't finish, but once you have addressed them, I'll go over it myself and do some final polishing.

**Comment by shenvitor** [comment]
https://github.com/ComPWA/compwa.github.io/actions/runs/9775660985/job/26986541402#step:5:865

for the record: link broken

**Comment by redeboer** [comment]
Yeah Launchpad is broken, happens sometimes

File: docs/report/033.ipynb
Comment (shenvitor) [inline docs/report/033.ipynb:1]: Interesting! So change to `jax.numpy` instead of `numpy` directly to increase computing speed (?)

Comment (shenvitor) [inline docs/report/033.ipynb:1]: I see... the bunch size was the same as the size before. Now we changed the bunch size to be way larger than event size

Comment (redeboer) [reply docs/report/033.ipynb:1]: I'm not sure yet, still slow

Comment (redeboer) [reply docs/report/033.ipynb:1]: I'm not sure yet, still slow

**Review by redeboer — APPROVED** [review]
Ready to merge 🚙

**Pull request merged**